controls/SV-204396.rb

```diff
@@ -6,14 +6,16 @@
     The session lock is implemented at the point where session activity can be determined.
     Regardless of where the session lock is determined and implemented, once invoked, the session lock must remain in
     place until the user reauthenticates. No other activity aside from reauthentication must unlock the system.'
-  desc 'check', %q(Verify the operating system enables a user's session lock until that user re-establishes access
-    using established identification and authentication procedures. The screen program must be installed to lock
-    sessions on the console.
-    Note: If the system does not have GNOME installed, this requirement is Not Applicable.
-    Check to see if the screen lock is enabled with the following command:
-    # grep -i lock-enabled /etc/dconf/db/local.d/*
-    lock-enabled=true
-    If the "lock-enabled" setting is missing or is not set to "true", this is a finding.)
+  desc 'check', %q(Verify the operating system enables a user's session lock until that user re-establishes access using established identification and authentication procedures.
+
+Note: If the system does not have GNOME installed, this requirement is Not Applicable.
+
+Check to see if the screen lock is enabled with the following command:
+
+     # grep -ir lock-enabled /etc/dconf/db/local.d/ | grep -v locks
```

Comment:
This update is functionally the same. However, this command in a sample rhel7 box (using Vagrant box iamseth/rhel-7.3) returns an empty string, while the InSpec command below that uses the gsettings API to get dconf settings gives the "true" response. The InSpec logic below uses the gsettings API for this check rather than checking the dconf back end directly as specified in the check text.

Should there be a describe.one that checks in both ways?

Reply:
Other controls that also use `gsettings`:

- [ ] `SV-204396`
- [ ] `SV-204398`
- [ ] `SV-204399`
- [ ] `SV-204400`
- [ ] `SV-204402`
- [ ] `SV-204403`
- [ ] `SV-204404`
- [ ] `SV-204456`
- [ ] `SV-214937`

Reply:
Established that this is a baseline issue and not a delta issue so [logged it](https://github.com/mitre/redhat-enterprise-linux-7-stig-baseline/issues/190) to address later.